### DISCUSSION TITLE
dbatools Update-DbaInstance has an issue.

### DISCUSSION THREAD

**Discussion by IgorB-PlanetArt** [dataplat/dbatools#8787] [Contributors]
I could not use update-dbainstance to push out a security patch (KB5021126).
Here is my statement:

cls
Update-DbaInstance `
        -ComputerName "ValidServerName.Domain" `
        -KB "5021126" `
        -Path "\\ValidServerName.Domain\s$\SQLServer2017-KB5021126-x64.exe" `
        -ExtractPath "\\ValidServerName.Domain\s$" `
        -Confirm:$false

I had tried it without -Path and\or -ExtractPath.
One of the errors I got was talking about a null array for Test-NetworkPath
Also got some warnings:
WARNING: [11:36:43][Update-DbaInstance] Explicit -Credential might be required when ru
nning agains remote hosts and -Path is a network folder
WARNING: [11:36:51][Get-SQLInstanceComponent] Component @Sharepoint on ServerName.Domain has a
n unknown version and was ommitted from the instance list
WARNING: [11:36:52][Resolve-DbaBuild] No info in store for this Release

**Comment by potatoqualitee** [comment]
this will likely be resolved once i merge @MikeyBronowski's changes, it's just too new and dbatools is currently lagging in development due to time constraints from me

**Comment by potatoqualitee** [comment]
Hello, can you please run the the following command:
```
Install-Module dbatools -AllowPrerelease -Scope CurrentUser
```

then restart powershell, and try again. 

this is prerelease2 of dbatools 2.0, which contains a ton of fixes
